lib/services/events/analyzers/google-analytics.js

```diff
@@ -18,6 +18,7 @@ module.exports = options => {
 
 Analyzer.prototype = {
 	prepare(payload) {
+		payload = payload.attributes;
```

Comment:
What would be the correct way to do since the SDK are sending using JSON API Service ?